addons/web/static/src/js/libs/bootstrap.js

```diff
@@ -24,4 +24,8 @@ $.fn.tooltip.Constructor.prototype.show = function () {
 
     return bootstrapShowFunction.call(this);
 };
+
+/* Bootstrap popover defaults overwrite */
+$.fn.popover.Constructor.Default.sanitize = false;
```

Comment:
Normally the default options for tooltips are automatically defaults options for popovers... but maybe not if we define the default here like that indeed. If this fixes something, this is probably the right fix for now (https://github.com/odoo/odoo/commit/36caf5fc16b5a95a0bf37f08f0caf974683b2df6)